File: assessment-frontend/src/app/util/operator.util.ts
import { Operator } from "../models/operator";

export function convertToSymbol(operator: Operator): string {
    switch(operator) {
        case Operator.ADD:
            return '+';
        case Operator.SUBTRACT:
            return '-';
        case Operator.MULTIPLY:
            return 'X';
        case Operator.DIVIDE:
            return '/';
    }
}

File: assessment-backend/src/main/java/aegon/hypotheken/assessment/util/OperatorUtil.java
package aegon.hypotheken.assessment.util;

import aegon.hypotheken.assessment.model.Operator;

public class OperatorUtil {

    private OperatorUtil() {}

    public static String getSymbol(Operator operator) {
        return switch (operator) {
            case ADD -> "+";
            case SUBTRACT -> "-";
            case MULTIPLY -> "X";
            case DIVIDE -> "/";
        };
    }
}
